Sources_v3/StreamChatUI/ChatChannel/MessageComposerSendButton.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class MessageComposerSendButton: ChatSquareButton {
+    // MARK: Underlying types
+    
+    public enum Mode {
+        case new, edit
+    }
+    
+    // MARK: - Properties
+    
+    public var mode: Mode = .new {
+        didSet {
+            updateContent()
+        }
+    }
+    
+    // MARK: - Overrides
+    
+    override open var isEnabled: Bool {
+        didSet {
+            updateContent()
+        }
+    }
+    
+    override open func updateContent() {
+        switch mode {
+        case .new:
+            setImage(UIImage(named: "sendMessageArrow", in: .streamChatUI), for: .normal)
+            transform = isEnabled ? CGAffineTransform(rotationAngle: -CGFloat.pi / 2.0) : .identity
+            tintColor = isEnabled ? .systemBlue : .sendButtonTint
```

Comment:
This feels very hacky, we shouldn't change the tint color of the button. It's the other way around - we need respect the tint color of the button! It seems what you want to do instead is set a different color for the image for the `.disabled` state?